### PR TITLE
SG-1324 - make campaign top fact images more uniform

### PR DESCRIPTION
**Pull request by aekong**

### CODE REVIEW

**Comment by zakiya** [comment]
I think this looks good. I tested on Chrome, Firefox, Safari - all screen sizes with two rows of facts.

File: web/themes/custom/sfgovpl/src/sass/paragraphs/_paragraph-facts.scss [hunk 1]
```diff
@@ -52,51 +52,32 @@
...
+        strong {
+          font-weight: $fw-medium;
         }
       }
     }
   }
-}
+  @include media($medium-screen) {
+    display: grid;
+    grid-template-columns: 32% 32% 32%;
```
Comment (zakiya) [inline web/themes/custom/sfgovpl/src/sass/paragraphs/_paragraph-facts.scss:79]: try `grid-template-columns: 1fr 1fr 1fr;` instead

It should work with IE.

Comment (aekong) [reply web/themes/custom/sfgovpl/src/sass/paragraphs/_paragraph-facts.scss]: thx! updated

Comment (aekong) [reply web/themes/custom/sfgovpl/src/sass/paragraphs/_paragraph-facts.scss]: @zakiya the `1fr` unit gives me this on certain window sizes (i think it is accommodating for word wrapping):
Screen Shot 2021-06-29 at 1 24 58 PM

`32%` seems more what we want? seems to be a bit more consistent:
Screen Shot 2021-06-29 at 1 25 10 PM

Comment (zakiya) [reply web/themes/custom/sfgovpl/src/sass/paragraphs/_paragraph-facts.scss:86]: What's the url? If 32% works, I'm not against it but I've never had to specify a percent when using grid before.

Comment (aekong) [reply web/themes/custom/sfgovpl/src/sass/paragraphs/_paragraph-facts.scss:86]: sorry: https://pr-933-sfgov.pantheonsite.io/covid19youthservices

**Review by zakiya — APPROVED** [review]

**Pull request merged**